.github/workflows/release.yml

```
@@ -0,0 +1,13 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    uses: guardian/gha-scala-library-release-workflow/.github/workflows/reusable-release.yml@use-java-21-lts
```

Whoops, my fault - should be:
```suggestion
    uses: guardian/gha-scala-library-release-workflow/.github/workflows/reusable-release.yml@main
```